icu.lua

```diff
@@ -87,7 +87,7 @@ project "icu"
     "icu4c/source/common/utrie2.cpp",
     "icu4c/source/common/uvector.cpp",
     "icu4c/source/common/wintz.cpp",
-    --"icu4c/source/data/in/icudt63l_dat.cpp",
+    "icu4c/source/data/icudt63_dat.c",
```

Comment:
do we need a 32bit version of this for our 32bit builds?

Reply:
The data only has endianess as a configuration so I think it is architecture agnostic.